### ISSUE TITLE
Improve editing experience of Latex preamble

### ISSUE THREAD

**Issue by lorenzhs** [otfried/ipe#261]
I'm using the Linux version of ipe, not sure if that makes a difference.

Currently, the Latex preamble is a bit of a pain to edit. There's no "Editor" button as for text elements, so I copy its contents into Emacs and back a lot. Additionally, resizing the window vertically increases the spacing between the various properties displayed below the box just as much as the box itself, which isn't overly useful:

![Screenshot](https://user-images.githubusercontent.com/142205/64409713-b5c20880-d089-11e9-89f4-c4f2eec09850.png)

From a brief look at the code, I think this is a bug as there is a call to `d:setStretch` for a specific row and column, but the Qt bindings don't touch `iRowStretch` at all? GTK does use it but my lipipeui.so is linked against Qt, not GTK.

**Comment by otfried** [comment]
I'll add an Editor button and will look at the row stretch problem.

When your document has a long and complicated preamble, it can be useful to put it in a separate style sheet, e.g. like this:
```
<ipestyle name="mypreamble">
<preamble>
% here goes the preamble
</preamble>
</ipestyle>
```

Save it as `mypreamble.isy` in the same directory as your document, and add it to the document (using Style sheets in the Edit menu).   

You can then keep the style sheet open in Emacs.   Every time you modify it in Emacs, press Shift+Ctrl+U (for "Update") in Ipe, and Ipe will automatically reload the style sheet.

**Comment by otfried** [comment]
Implemented in 7.2.13.

**Comment by lorenzhs** [comment]
Thanks for implementing it and the useful tip!
